Test/civl/samples/ticket.bpl

```diff
@@ -10,8 +10,8 @@ type {:linear "tid"} X;
 const nil: X;
```

Comment:
I think it would be nice to replace this with `Option<_>` from the library. (And rename `X` to `Tid`.)

Reply:
Good idea. Will do.